resources/user-agents/browsers/chrome/chrome-11.json

```diff
@@ -29,119 +29,79 @@
       },
       "children": [
         {
-          "match": "Mozilla/5.0 (X11; Linux*; HTC/Sensation/*) applewebkit* (*khtml*like*gecko*) Chrome/#MAJORVER#.*Safari/*",
-          "device": "HTC Z710",
-          "platforms": [
-            "Android_4_0"
-          ]
-        },
-        {
-          "match": "Mozilla/5.0 (X11; Linux x86_64*; HTC/Sensation/*) applewebkit* (*khtml*like*gecko*) Chrome/#MAJORVER#.*Safari/*",
```

Comment:
This one is a near dupe of the one above, just has the "x86_64" bit added.  Since there was no platform change or anything with that addition, I felt it was unnecessary to have both of these, since the one above should work for both here.